lib/fields.ts

```diff
@@ -0,0 +1,70 @@
+// Helper functions for creating fields
+
+import { FieldDefinition, ParquetType, SchemaDefinition } from "./declare";
+
+export function createStringField(optional = true, fieldOptions: FieldDefinition = {}): FieldDefinition {
+    return { ...fieldOptions, optional, type: 'UTF8' };
+}
+
+export function createBooleanField(optional = true, fieldOptions: FieldDefinition = {}): FieldDefinition {
+    return { ...fieldOptions, optional, type: 'BOOLEAN' };
+}
+
+export function createIntField(size: 32 | 64, optional = true, fieldOptions: FieldDefinition = {}): FieldDefinition {
```

Comment:
What about `INT96`? 
 --> https://parquet.apache.org/docs/file-format/types/

Reply:
Good call out. I checked and according to the Parquet Thrift INT96 is "only used for legacy timestamps".